sdk/rust-examples/prime-numbers-generator/README.markdown

```diff
@@ -0,0 +1,30 @@
+# Prime Numbers Generator
+
+This program provides an example of executing multiple programs sequentially inside Veracruz `freestanding-execution-engine`, where each program's input depends on the previous' output. This is similar to the Linux pipeline `command1 | command2 | command3`. 
+
+This is a two-step program where the first step provides its output to the second program. The final output is the prime numbers between 2 and `upper_limit`, which you will specify . This is an implementation of the [Sieve of Eratosthenes](https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes) algorithms. 
+
+- `generate-set`: this program generates all numbers from 2 to `upper_limit` 
```

Comment:
Should we move `generate-set` to `sdk/data-generators` (https://github.com/veracruz-project/veracruz/tree/main/sdk/data-generators) and add it to the Makefile (https://github.com/veracruz-project/veracruz/blob/main/workspaces/shared.mk)?

Reply:
I think this example may be different (?) . Regarding the second point in #377 (pipeline example), what I understood is that each program shall provide an output which will be consumed as an input by the next program inside the FEE.  Hence, I made the dataset to be generated inside the FEE and consumed as an input by the next program; `keep-primes` . please correct me if I misunderstood the requirements.

Reply:
Makes perfect sense :+1: